examples/siro_sandbox/server/keyframe_utils.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Any
+
+
+def update_consolidated_keyframe(consolidated_keyframe, inc_keyframe):
```

Comment:
I don't like this whole approach of doing detailed gfx-replay keyframe logic in Python, in the HITL codebase. This is hacky. Hopefully we'll clean up the HITL codebase in the future.